test/literature/live/blog_live_test.exs

```diff
@@ -169,6 +175,38 @@ defmodule Literature.BlogLiveTest do
                )
       end
     end
+
+    test "post language tags only renders equal to x-default", %{
+      conn: conn,
+      publication: publication,
+      post: post
+    } do
+      {:ok, _view, html} = live(conn, Routes.literature_path(conn, :show, post.slug))
+
+      Literature.update_publication(publication, %{ex_default_locale: "de"})
+
+      current_url = @endpoint.url() <> Routes.literature_path(conn, :show, post.slug)
+
+      assert get_element(
+               html,
+               "link[href='#{current_url}'][hreflang='en'][rel='alternate']"
+             )
+
+      assert get_element(
+               html,
+               "link[href='#{current_url}'][hreflang='x-default'][rel='alternate']"
+             )
+
+      assert get_element(
```

Comment:
@rcmonsayac im not sure i follow why one  link is refuted and one is asserted when link seems to be the same am i missing something ?

Reply:
@dgross881 Fixed tests